injective_functions/snapshot/__init__.py

```
@@ -0,0 +1,17 @@
+import json
```

Ruff (0.8.0)

`json` imported but unused

(F401)